docs/Context_Binding_(Element_Binding)_91f05e8.md

```diff
@@ -226,16 +229,11 @@ Say we want to display companies in a `sap.m.List` control. Here’s what the XM
 ``` xml
 <mvc:View
 	controllerName="sap.ui.sample.App"
-	xmlns="sap.m"
-	xmlns:mvc="sap.ui.core.mvc">
-	  <List id=”companyList” items="{companies}">
-		<items>
-			<StandardListItem
-	 	title="{name}"
-	 	description="{city}"
-			/>
-		 </items>
-	  </List>
+	xmlns:mvc="sap.ui.core.mvc"
```

Comment:
@boghyon any reason why you prefer to put the default namespace not as the first namespace?

Reply:
I namespaces are added in the order of the respective controls in the XML definition. So, `xmlns:mvc="sap.ui.core.mvc"` happens to be always the first one (`<mvc:View ...>`) and `sap.m` at some point later.